code/style.css

```diff
@@ -1,4 +1,258 @@
 html {
-  background: #0025ff;
-  color: #fff;
+  background: #ffe2e2;
+}
+
+/* HEADER SECTION*/
+
+.navbar {
+  background-color:#bad7df);
+  margin: 0px;
+  padding-bottom: 0%;
+  max-width: 100%;
+}
+
+.titlelogo {
+  display: flex;
+  justify-content: space-evenly;
+}
+
+.logo {
+  width: 50px;
+}
+
+.main-heading {
+  font-family: "Permanent Marker", cursive;
+  font-weight: 300;
+  font-size: 30px;
+  margin: 0px;
+  padding: 12px 0px 0px 12px;
+}
+
+.nav-menu {
+  display: flex;
+  justify-content: space-evenly;
+  flex-wrap: wrap;
+  width: 100%;
+  font-family: "IBM Plex Sans", sans-serif;
+}
+
+/* BIG NEWS SECTION */
+
+.big-news {
+  display: flex;
+  flex-direction: column;
+  align-content: center;
+  max-width: 100%;
+  margin-top: 20px;
+  margin-bottom: 20px;
+  background-color: #bad7df;
+  padding: 40px;
+  border: 6px dotted darkmagenta;
+}
+
+.big-news img {
+  width: 100%;
+  height: auto;
+}
+
+.big-news-heading {
+  font-family: "IBM Plex Sans", sans-serif;
+  font-size: 25px;
+  margin-left: 15px;
+}
+
+.big-news-subheading {
+  font-family: "IBM Plex Sans", sans-serif;
+  font-size: 20px;
+  margin-left: 15px;
+}
+
+.big-news-text {
+  font-family: "IBM Plex Sans", sans-serif;
+  font-size: 18px;
+  text-align: center;
+}
+
+/*HORISONTAL RULE*/
+.hr1 {
+  border-width: 8px 0px 0px 0px;
+  border-color: #99ddcc;
+  border-radius: 10%;
+  margin-bottom: 20px;
+}
+
+/* NEWS SCROLL SECTION*/
+
+.news-scroll {
+  display: flex;
+  flex-direction: column;
+  width: 100%;
+  align-content: space-around;
+  margin-bottom: 10px;
+}
+
+.news-card {
+  /* display: flex; */
+  color: darkcyan;
+  background-color: #bad7df;
+  font-family: Verdana, Geneva, Tahoma, sans-serif;
+  padding: 20px 30px;
+  border: 3px solid darkmagenta;
+  margin: 20px;
```

Comment:
The margin is also creating a small horizontal scroll because it makes the content too wide:


<img width="333" alt="Skärmavbild 2021-02-03 kl  09 59 39" src="https://user-images.githubusercontent.com/51529201/106722689-89a41180-6606-11eb-8b72-019b00f58986.png">